bindings/java/hyperic_jni/src/org/hyperic/jni/ArchNameTask.java

```diff
@@ -75,7 +75,7 @@ else if (ArchLoader.IS_AIX) {
         if (ArchName.is64()) {
             getProject().setProperty("jni.arch64", "true");
             if (ArchLoader.IS_LINUX) {
-                if (!osArch.equals("ia64")) {
+                if (!osArch.equals("ia64") && osArch.equals("arm64")) {
```

Comment:
It is required to build Sigar on aarch64 otherwise build is failing with below error:
"gcc: error: unrecognized command line option '-m64'"
I changed to this line as " if (!osArch.equals("ia64") && !osArch.equals("aarch64")){"